src/app/core/auth.service.ts

```diff
@@ -299,14 +300,14 @@ export class AuthService {
   }
 
   /**
-   * Identifies if the current logged in user has reviewer priviledges.
+   * Identifies if the current logged in user has reviewer privileges.
```

Comment:
❤️

Reply:
The funny part is I misspelled the same word in my commit message